packages/math-input/src/data/key-configs.ts

```diff
@@ -4,45 +4,47 @@
 import * as i18n from "@khanacademy/wonder-blocks-i18n";
 
 import {DecimalSeparator, IconType, KeyType} from "../enums";
+import {KeyConfig} from "../types";
 import {decimalSeparator} from "../utils";
 
 import Keys from "./keys";
 
-export type KeyConfig = {
-    id: string;
-    type: string;
-    ariaLabel: string;
-};
-
 // I tried to make the below {[key in Keys]: KeyConfig}
 // but we are doing all kinds of sneaky magic that makes it hard to
 // type this safely. Leaving it for now as a generic index signature.
-const KeyConfigs: {[key: string]: any} = {
+const KeyConfigs: {
+    [key in Keys]: KeyConfig;
```

Comment:
if this shouldn't be exhaustive, I can change this to `[key in Keys]?: KeyConfig;` and restore some of that dynamic stuff.